testutil/testclient/testsupplier/client.go

```diff
@@ -6,15 +6,14 @@ import (
 
 	"cosmossdk.io/depinject"
 	"github.com/golang/mock/gomock"
-	"github.com/pokt-network/smt"
-	"github.com/stretchr/testify/require"
-
 	"github.com/pokt-network/poktroll/pkg/client"
 	"github.com/pokt-network/poktroll/pkg/client/supplier"
 	"github.com/pokt-network/poktroll/pkg/client/tx"
 	"github.com/pokt-network/poktroll/testutil/mockclient"
 	"github.com/pokt-network/poktroll/testutil/testclient/testtx"
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	"github.com/pokt-network/smt"
+	"github.com/stretchr/testify/require"
```

Comment:
Same as https://github.com/pokt-network/poktroll/pull/253/files#r1423131221 maybe you should look into using `goimports-reviser` it should fix this sort of thing automatically for you